code/game/machinery/medipen_refiller.dm

```diff
@@ -0,0 +1,81 @@
+/obj/machinery/medipen_refiller
+	name = "Medipen Refiller"
+	desc = "A machine that refills used medipens with chemicals, it needs them first."
+	icon = 'icons/obj/machines/medipen_refiller.dmi'
+	icon_state = "medipen_refiller"
+	density = TRUE
+	circuit = /obj/item/circuitboard/machine/medipen_refiller
+	idle_power_usage = 100
+	/// list of medipen subtypes it can refill
+	var/list/allowed = list(/obj/item/reagent_containers/hypospray/medipen = /datum/reagent/medicine/epinephrine,
```

Comment:
don't epipens have formaldehyde in them now?

Reply:
3 u of  magic formaldehyde or ugly list inside a list